projects/valtimo/components/src/lib/components/view-content/type-converters/booleanTypeConverter.ts

```diff
@@ -32,6 +37,8 @@ export class BooleanTypeConverter implements TypeConverter {
         : enumeration[Object.keys(enumeration)[0]] || 'No';
     }
 
-    return value ? 'Yes' : 'No';
+    return value
+      ? this.translateService.instant('booleanTypeConverter.Yes')
+      : this.translateService.instant('booleanTypeConverter.No');
```

Comment:
```suggestion
    return this.translateService.instant(`booleanTypeConverter.${value ? 'Yes' : 'No'}`)
```

Reply:
Also, this will break the boolean view type in the CarbonList...make sure to account for that